build.sbt

```diff
@@ -130,6 +130,6 @@ val commonSettings = Seq(
     }
   },
   scalaTestVersion := "3.2.16",
-  poiVersion := "5.2.2",
+  poiVersion := "5.2.3",
```

Comment:
Look at doing the following, that should bring in the changes needed for pre-Spark 3.4:

Line 26, add the following: `val log4JVersion = settingKey[String]("Apache Log4J version")`

Line 65, add the following to the collection

```
  "org.apache.logging.log4j" % "log4j-core" % log4JVersion.value % Compile,
  "org.apache.logging.log4j" % "log4j-api" % log4JVersion.value % Compile
```

Line 133: Add this after `poiVersion`: `log4JVersion := "2.20.0",`

Reply:
Excellent, with that update we bring log4j as a project dependency so we don't get conflicts across spark versions, pushed the change request.